semanticz-connector-neptune/src/main/java/zone/cogni/semanticz/connectors/neptune/NeptuneSparqlService.java

```diff
@@ -0,0 +1,131 @@
+package zone.cogni.semanticz.connectors.neptune;
+
+import org.apache.jena.query.*;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdfconnection.RDFConnection;
+import org.apache.jena.rdfconnection.RDFConnectionRemote;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import zone.cogni.semanticz.connectors.general.SparqlService;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.function.Function;
+
+public class NeptuneSparqlService implements SparqlService {
```

Comment:
There is also `RDFConnectionSparqlService` - would is be reusable by chance? We can even adjust it/if needed?

Reply:
seems already extendable with no overhead